src/kani-compiler/rustc_codegen_kani/src/codegen/intrinsic.rs

```diff
@@ -519,6 +519,7 @@ impl<'tcx> GotocCtx<'tcx> {
             "volatile_copy_memory" => codegen_intrinsic_copy!(Memmove),
             "volatile_copy_nonoverlapping_memory" => codegen_intrinsic_copy!(Memcpy),
             "volatile_load" => self.codegen_expr_to_place(p, fargs.remove(0).dereference()),
+            "volatile_store" => self.codegen_volatile_store(instance, intrinsic, fargs, loc),
```

Comment:
When @danielsn looked at this recently, he suggested adding the following assert:
```rust
assert!(self.place_ty(p).is_unit());
```

Reply:
Added.